scripts/environment/bootstrap-ubuntu-20.04.sh

```diff
@@ -137,16 +145,16 @@ tar \
 cp "${TEMP}/${MOLD_TARGET}/bin/mold" /usr/bin/mold
 
 # Set Cargo to use mold as its linker.
-CARGO_OVERRIDE_DIR="${HOME}/.cargo"
-mkdir -p "${CARGO_OVERRIDE_DIR}"
+CARGO_BIN_DIR="${CARGO_OVERRIDE_DIR}/bin"
+mkdir "$CARGO_BIN_DIR"
 
-CARGO_OVERRIDE_CONF="${CARGO_OVERRIDE_DIR}/config.toml"
-cat <<EOF >>"${CARGO_OVERRIDE_CONF}"
-[target.x86_64-unknown-linux-gnu]
-rustflags = ["-D", "warnings", "-C", "linker=clang", "-C", "link-arg=-fuse-ld=/usr/bin/mold"]
+RUST_WRAPPER="${CARGO_BIN_DIR}/wrap-rustc"
+cat <<EOF >"$RUST_WRAPPER"
+#!/bin/sh
+exec mold -run "$@"
```

Comment:
I think you'll need to `chmod +x "$RUST_WRAPPER"` so it can actually be executed?